lua/nvim-tree/utils.lua

```diff
@@ -38,6 +38,14 @@ function M.path_split(path)
   return path:gmatch("[^" .. path_separator .. "]+" .. path_separator .. "?")
 end
 
+--- Retrieve last element of a path.
+--- @param path string Path to retrieve last element from.
+--- @return string Last element of the path.
+function M.get_last_path_elem(path)
```

Comment:
Could we please use `fnamemodify( , ":t")`

Reply:
Will do!